src/busco/busco/config.vsh.yaml

```
@@ -0,0 +1,215 @@
+functionality:
+  name: busco
+  description: Assessment of genome assembly and annotation completeness with single copy orthologs
+  info:
+    keywords: [Genome assembly, quality control]
+    homepage: https://busco.ezlab.org/
+    documentation: https://busco.ezlab.org/busco_userguide.html
+    repository: https://gitlab.com/ezlab/busco
+    reference: "10.1007/978-1-4939-9173-0_14"
+    licence: MIT
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: --input
+          alternatives: ["-i"]
+          type: file
+          description: |
+            Input fasta file or directory containing input fasta files to analyse. Fasta files can either be a nucleotide or protein fasta file, depending on the BUSCO mode.
+          required: true
+          example: file.fasta
+        - name: --mode
+          alternatives: ["-m"]
+          type: string
+          choices: ["protein", "genome", "transcriptome"]
+          required: true
+          description: |
+            Busco assesment mode
+          example: protein
+        - name: --lineage_dataset
+          alternatives: ["-l"]
+          type: string
+          required: false
+          description: |
+            Specify a BUSCO lineage dataset that is most closely related to the assembly or gene set being assessed. 
+            The full list of available datasets can be viewed using the "busco_list_datasets" component.
```

This component doesn't exist yet. I have an alternative proposal:

```suggestion
            The full list of available datasets can be viewed [here](https://busco-data.ezlab.org/v5/data/lineages/) or by running `busco --list-datasets` (which requires installing the tool).